packages/internals/src/__tests__/getSchema.test.ts

```diff
@@ -114,3 +161,25 @@ it('finds the conventional schema path with yarn workspaces', async () => {
     `"src/__tests__/__fixtures__/getSchema/conventional-path-workspaces/packages/b/schema.prisma"`,
   )
 })
+
+it('fails with no schema in workspaces', async () => {
+  const res = await testSchemaPath('no-schema-workspaces')
+
+  expect(res).toMatchInlineSnapshot(`
+    [Error: Could not find a schema.prisma file that is required for this command.
+    You can either provide it with --schema, set it as \`prisma.schema\` in your package.json or put it into the default location.
+    Checked following paths:
+
+    schema.prisma: file not found
```

Comment:
Yarn stuff, printing all the paths where we looked.

Reply:
Ah, now I know why you stumbled over this today :D